frontend/src/components/pages/Landing.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { useHistory } from "react-router-dom";
+import {
+  Box,
+  Center,
+  Text,
+  Stack,
+  VStack,
+  Image,
+  Button,
+  Link,
+} from "@chakra-ui/react";
+import * as Routes from "../../constants/Routes";
+
+import JumpMathLogo from "../../assets/jump-math-logo.png";
+
+const PageLinks = (): React.ReactElement => {
+  const history = useHistory();
+  const navigateTo = () => history.push(Routes.LOGIN_PAGE);
+
+  return (
+    <VStack gap={1.5}>
+      <Button onClick={navigateTo} width={300} variant="primary">
+        Admin
+      </Button>
+      <Button onClick={navigateTo} width={300} variant="primary">
+        Teacher
+      </Button>
+      <Button onClick={navigateTo} width={300} variant="primary">
+        Student
+      </Button>
+    </VStack>
+  );
+};
+
+const Landing = (): React.ReactElement => {
+  return (
+    <Box>
+      <Stack p={8} pb={20} justifyContent="left">
+        <Link textStyle="subtitle2" href="https://jumpmath.org/">
```

Comment:
Would usually use `<RouterLink>` here, but `react-router` does not support routing to external links without opening a new tab.